src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    static final int FIGURE_ARRAY_SIZE = 6;
+
+    public static void main(String[] args) {
```

Comment:
The `main` method should be as clean as possible. Consider moving the logic of creating and drawing figures to a separate method or class to adhere to the Single Responsibility Principle.